controllers/replication.storage/volumereplication_controller.go

```diff
@@ -402,9 +402,11 @@ func (r *VolumeReplicationReconciler) Reconcile(ctx context.Context, req ctrl.Re
 	return ctrl.Result{}, nil
 }
 
-// getScheduleTime takes parameters and returns the scheduling interval
-// after converting it to time.Duration. If the schedulingInterval is empty
-// or there is error parsing, it is set to the default value.
+// getScheduleTime takes parameters and returns the half of the scheduling
+// interval after converting it to time.Duration. The interval is converted
+// into half to keep the LastSyncTime and Storage LastSyncTime to be in sync.
+// If the schedulingInterval is empty or there is error parsing, it is set to
+// the default value.
```

Comment:
(partial nit) There is a bit of confusion with the function naming and the current return value. I was a little surprised that `getScheduleTime` returns 1/2 the schedule time, and checked the code to see that it was only called to set the reconcile interval and not to get the schedule time in general.

So function reuse anywhere else could be inappropriate. Can we rename this function to `reconcileInterval`, or `infoReconcileInterval` to disambiguate?

Reply:
yes correct, i should have changed the function name.